components/feature/pwa/src/main/java/mozilla/components/feature/pwa/WebAppShortcutManager.kt

```diff
@@ -166,6 +165,8 @@ class WebAppShortcutManager(
         val shortLabel = manifest.shortName ?: manifest.name
         storage.saveManifest(manifest)
 
+        emitPwaInstallFact()
```

Comment:
Both emit calls should be in `requestPinShortcut` as that corresponds to installing/pinning the PWA.